Services/data/devicemodels/cab-01-protobuf.json

```diff
@@ -0,0 +1,51 @@
+{
+  "SchemaVersion": "1.0.0",
+  "Id": "cab-01-proto",
+  "Version": "0.0.1",
+  "Name": "Cab 01 Proto",
+  "Description": "Cab data on Protobuff",
+  "Protocol": "AMQP",
+  "Simulation": {
+    "InitialState": {
+      "online": true,
+      "latitude": 47.445301,
+      "longitude": -122.296307,
+      "speed": 30,
+      "heading": 1,
+      "altitude": 1000,
+      "accuracy": 90
+    },
+    "Interval": "00:00:10",
+    "Scripts": [
+      {
+        "Type": "javascript",
+        "Path": "cab-01-protobuf-state.js"
+      }
+    ]
+  },
+  "Properties": {
+    "Type": "Cab01Proto",
+    "Location": "Field",
+    "Latitude": 47.445301,
+    "Longitude": -122.296307
+  },
+  "Telemetry": [
+    {
+      "Interval": "00:00:10",
+      "MessageTemplate": "{\"latitude\":${latitude},\"longitude\":${longitude},\"speed\": ${speed},\"heading\": ${heading},\"altitude\":${altitude},\"accuracy\":${accuracy}}",
+      "MessageSchema": {
+        "Name": "GPSPacket;v1",
```

Comment:
for Protobuf we need a place to store the class name to instantiate the object later in `DeviceClient`. Add a new `ClassName` field here (and relevant classes):
```
"ClassName": "Microsoft.Azure.IoTSolutions.DeviceSimulation.Protobuf.Models.GPSPacket",
```